.gitignore

```diff
@@ -38,8 +38,7 @@ sitemap.xml
 sitemap-*.xml
 
 # environmonet
-# .env
-.env.local
```

Comment:
Shouldn't we keep `.env.local` in the ignore?

Reply:
I removed that `.env.local` because it has already been mentioned above on line 27